src/configs/cmake.rs

```diff
@@ -0,0 +1,18 @@
+use crate::config::{ModuleConfig, RootModuleConfig};
+
+use starship_module_config_derive::ModuleConfig;
+
+#[derive(Clone, ModuleConfig)]
+pub struct CMakeConfig<'a> {
+    pub format: &'a str,
+    pub disabled: bool,
+}
+
+impl<'a> RootModuleConfig<'a> for CMakeConfig<'a> {
+    fn new() -> Self {
+        CMakeConfig {
+            format: "via [🛆 $version](blue bold) ",
+            disabled: false,
+        }
+    }
+}
```

Comment:
As is discussed in https://github.com/starship/starship/discussions/1116#discussioncomment-6579, It is recommended to have `symbol` and `style` options for historical reasons.